### ISSUE TITLE
Esc Key does not work when focused on map context menu item: "Copy Coordinates (C)"

### ISSUE THREAD

**Issue by AliyanH** [Maps4HTML/Web-Map-Custom-Element#746]
When map context menu is opened (Shift+F10 or right click), and you tab into the "Copy Coordinates (C)" map contextmenu item, pressing esc does not dismiss the contextmenu as it does for all the other contextmenu items.
